inventories/group_vars/control-plane/common.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 metal_control_plane_provider_tenant: metal-stack
-metal_control_plane_host_provider: vagrant
+metal_control_plane_host_provider: kind
```

Comment:
I think this variable can even be removed. Not sure anymore but it seems like very old to me (probably back in those days when we provisioned K8s through Ansible :see_no_evil: ).

Reply:
According to metal-roles:
`| metal_control_plane_host_provider     |           | The name of the provider where the control plane is hosted, can be used for provider specific switches in the automation |`
Should I remove it from every place?

Reply:
Yes, feel free to remove it from metal-roles as well. When it's not used in the public roles, it is not required.